slicer_configurations/README.md

```diff
@@ -28,4 +28,5 @@ like so:
 | hijax_pl_ | [Simplify3D](./Simplify3D/hijax_pl/beds) | Bed images for Simplify3D | :x: | :x: | :heavy_check_mark: | :x: |
 | hartk1213 | [PrusaSlicer V0 Bed and Textures](./PrusaSlicer/hartk1213/V0) | Bed STL and textures for Prusa Slicer | :heavy_check_mark: | :x: | :x: | :x: |
 | koonweee | [SuperSlicer v1.8 Bed](./SuperSlicer/koonweee/beds) | Bed STL for SuperSlicer | :x: | :heavy_check_mark: | :x: | :x: |
+| hoangnam123 | [SuperSlicer v2.4 Bed](./SuperSlicer/hoangnam123) | Bed STL for SuperSlicer | :x: | :x: | :heavy_check_mark: | :x: |
```

Comment:
Please move this line up so it's in alphabetical order.  I realize some lines above it are in the wrong order, but nevertheless please move your line up so it's before `koonweee`.

Reply:
i have fixed the order for the other entry also